tests_metricflow/query_rendering/test_query_rendering.py

```diff
@@ -603,3 +607,61 @@ def test_non_additive_dimension_with_non_default_grain(
         dataflow_plan_builder=dataflow_plan_builder,
         query_spec=query_spec,
     )
+
+
+@pytest.mark.sql_engine_snapshot
```

Comment:
Not for this PR, but maybe we should consolidate `tests_metricflow/integration/query_output/test_query_output.py` and `tests_metricflow/query_rendering/test_query_rendering.py`.